docs/tschemes.rst

```diff
@@ -0,0 +1,314 @@
+Time Schemes
+============
+.. _tschemes:
+
+MoorDyn 2 is deployed with several time schemes, with different features,
```

Comment:
"MoorDyn-C version 2 is deployed..."

Reply:
Fixed on https://github.com/FloatingArrayDesign/MoorDyn/pull/182/commits/1bb95b347bb7c25d949dfde5ce6b5e448a2a073a